src/main/resources/synthetic.xml

```diff
@@ -25,16 +25,30 @@
                   description="Absolute Path to 7zip.exe (only applicable on Windows). If provided this one will be used, else the prepackaged one will be used."/>
         <property name="envName" label="Environment Name" required="false"
                   description="Environment name to be used for forecast and deploy (e.g. XE)"/>
-
         <property name="username" label="Username" required="false" category="Credentials"
                   description="Database username when the project has the 'Specify database credentials at operation run time' option set." />
         <property name="password" label="Password" required="false" category="Credentials" password="true"
                   description="Database password when the project has the 'Specify database credentials at operation run time' option set." />
-        <method name="check"
+        <property name="audit_username" label="Audit Username" required="false" category="Credentials"
```

Comment:
Can we stick to using [camel case](https://en.wikipedia.org/wiki/Camel_case) for the property names to be consistent?

Reply:
Done. All properties in synthetic.xml are now camel case and refactored all files.